docs/install/create-project-cookieplone.md

```diff
@@ -12,41 +12,40 @@ myst:
 
 # Install Plone with Cookieplone
 
-This chapter describes how you can create a web application using the {term}`Cookieplone` template.
-
-This template is the recommended way to start a new Plone project using the Volto frontend.
+This chapter describes how you can create a web application using {term}`Cookieplone`.
+Cookieplone is the recommended way to create a Plone project as an add-on using the Volto frontend.
```

Comment:
"as an add-on" is confusing here. The project template is useful both for creating a Plone site to deploy and for creating an add-on to release.

I gave more thoughts in response to https://github.com/plone/cookieplone-templates/issues/191 but what about the following as a quick fix:

Cookieplone is the recommended way to create a Plone project using the Volto frontend.
These instructions can be used to start building a Plone site, or to develop an add-on that can be released and used as a part of other Plone sites.
The project they set up also includes tools for development and deployment.